development/kops/set_environment.sh

```diff
@@ -70,6 +64,7 @@ export PREFLIGHT_CHECK_PASSED
 VERSION=$(cat ${BASEDIR}/../../projects/kubernetes/kubernetes/${RELEASE_BRANCH}/GIT_TAG)
```

Comment:
Is this value being used in any script? Should it be exported in that case?

Reply:
Done. It was working because VERSION was still in scope, but better to export for consistency